Pages/navbar.html

```diff
@@ -25,16 +25,7 @@
                 <div
                     class="absolute left-0 top-full hidden group-hover:block bg-white border border-gray-300 text-center text-sm z-50"
                 >
-                    <a
-                        href="/index.html"
-                        class="block px-4 py-2 text-gray-900 hover:text-black"
-                        >Projects</a
-                    >
-                    <a
-                        href="/index.html"
-                        class="block px-4 py-2 text-gray-900 hover:text-black"
-                        >Leaders</a
-                    >
+
                 </div>
```

Comment:
Delete this whole div if the dropdown menu is being removed (it shows a little empty box when I hover over the home button right now)